src/vic3save/src/stats.rs

```diff
@@ -0,0 +1,223 @@
+use crate::Vic3Date;
+use serde::Deserialize;
+use std::collections::HashMap;
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct VicStatsChannel {
+    // Date of last sample
+    pub date: Vic3Date,
+    //  next free index (number of samples + 1)
+    pub index: u32,
+    // Value
+    pub values: Vec<f32>,
+}
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct Vic3CountryStats {
+    // Sample rate of ticks (sample rate of 28, works to 1 week)
+    pub sample_rate: u32,
+    // Not sure, double the index?
+    pub count: u32,
+    pub channels: HashMap<u32, VicStatsChannel>,
```

Comment:
Also looks like some saves lack channels, so this should be annotated with `#[serde(default)]`